packages/devextreme/js/__internal/scheduler/appointment_popup/m_form.ts

```diff
@@ -923,6 +928,22 @@ export class AppointmentForm {
     return buttons;
   }
 
+  private enableRecurrenceSettingsButton(): void {
```

Comment:
We don't need this method. We can just set `disabled: false` in [this line](https://github.com/sjbur/DevExtreme/blob/8575875aa6e544b158bbb49c45584b01b9e56c7d/packages/devextreme/js/__internal/scheduler/appointment_popup/m_form.ts#L913)